src/components/Book.js

```diff
@@ -18,44 +21,51 @@ const Book = (props) => {
     // Delete in the backend
     dispatch(removeBookAsync(id));
   };
-  const styles = {
-    content: {
-      display: 'flex',
-      flexDirection: 'row',
-      gap: '6px',
-    },
-    para: {
-      fontWeight: 'bold',
-    },
-  };
+
   return (
-    <div className="book-container">
-      <div className="content" style={styles.content}>
-        <span>{category}</span>
-        <p className="para" style={styles.para}>{title}</p>
-        <p>{author}</p>
-        <Button type="submit" dataId={id} handleClick={handleRemove}>
-          Remove
-        </Button>
-        <Button type="button" data-id={id}>
-          <span>Edit</span>
-        </Button>
-      </div>
-      <div>
-        <div>
-          <span>70%</span>
-          <span>Completed</span>
+    <article className="book-cont">
+      <section className="cat-sec">
+        <span className="category">{category}</span>
+        <h2 className="title">{title}</h2>
+        <p className="author">{author}</p>
+        <div className="interact-btns-cont">
+          <Button
+            className="interact-btns"
+            type="submit"
+            dataId={id}
+            handleClick={handleRemove}
+          >
+            Comment
+          </Button>
+          <Button
+            className="interact-btns"
+            type="submit"
+            dataId={id}
+            handleClick={handleRemove}
+          >
+            Remove
+          </Button>
+          <Button className="interact-btns" type="button" data-id={id}>
+            Edit
+          </Button>
         </div>
-      </div>
-      <div>
-        <span>Current Chapter</span>
-        <h4>Chapter 56</h4>
-        <button type="button" className="progress-btn">
-          Update progress
-        </button>
-      </div>
+      </section>
 
-    </div>
+      <section className="prg">
+        <Progress progress={randomPer} />
+      </section>
+
+      <section className="chapters-sec">
+        <div className="line" />
+        <div>
+          <span className="chapter-sec-head">Current Chapter</span>
+          <p className="chapter">Chapter 5</p>
+          <button type="button" className="update-progress">
+            Update progress
+          </button>
+        </div>
+      </section>
+    </article>
```

Comment:
- [ ] KIndly make sure that you make the buttons and texts on the left side of your page look exactly like the zephin design
- [ ] When I add more than one book the books are aligned on a row, please update your css so that every book will be displayed in a new line/block  and not in a row

Zephin Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_754](https://user-images.githubusercontent.com/69638013/224485588-d6319309-a4c3-40d9-8c70-1e0f2030fce2.png) | ![Screenshot_755](https://user-images.githubusercontent.com/69638013/224485961-72f40e74-7fac-4332-ba1c-5586b56b791a.png)